clickhouse_driver/util/helpers.py

```diff
@@ -1,12 +1,27 @@
 from itertools import islice, tee
 
+try:
+    import numpy as np
+
+    CHECK_NUMPY_TYPES = True
+except ImportError:
+    CHECK_NUMPY_TYPES = False
+
+
+def _check_sequence_to_be_an_expected_iterable(seq):
```

Comment:
Numpy/pandas helpers should be located in `clickhouse_driver/numpy/helpers.py`.

I'd keep two different versions of chunks: one for non-numpy and another one for numpy.